extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -262,7 +262,9 @@ export class LinkerManager {
     if (this.isDomainMatch_(hostname, domains)) {
       const linkerValue = createLinker(/* version */ '1',
           this.resolvedIds_[name]);
-      el.href = addParamToUrl(href, name, linkerValue);
+      if (linkerValue) {
```

Comment:
I understand you want the logic goes to `createLinker` and I agree this looks good now. 
But depending on if we want to support wildcard domain matching, I think it may worth moving the check before the domain match check.

Reply:
Sounds good. I will keep this in mind when we add this feature.